RELEASE-NOTES.md

```diff
@@ -95,7 +95,7 @@ This includes API changes we did not warn about since at least `3.11.0` (2021-01
 - New features for BART:
   - Added linear response, increased number of trees fitted per step [5044](https://github.com/pymc-devs/pymc3/pull/5044).
   - Added partial dependence plots and individual conditional expectation plots [5091](https://github.com/pymc-devs/pymc3/pull/5091).
-  - Modify how particle weights are computed. This improves accuracy of the modeled function (see [5177](https://github.com/pymc-devs/pymc3/pull/5177)).
+  - Modify PGBART sampler. Particles are not longer reweighted and the trees are reset from time to time to avoid getting trap in a local mnima. This improves accuracy of the modeled function and improves convergence (see [5223](https://github.com/pymc-devs/pymc3/pull/5223)).
```

Comment:
```suggestion
  - Modify PGBART sampler. Particles are not longer reweighted and the trees are reset from time to time to avoid getting trap in a local minima. This improves accuracy of the modeled function and improves convergence (see [5223](https://github.com/pymc-devs/pymc3/pull/5223)).
```